### ISSUE TITLE
AI GPI logic

### ISSUE THREAD

**Issue by randomnublet** [LoneGazebo/Community-Patch-DLL#6359]
3-15b

Look at crazy Elizabeth moving that Great Prophet all the way from London to the craphole of city that is Newcastle*

![civholy](https://user-images.githubusercontent.com/53142313/76765908-793d3b00-6797-11ea-94ca-dc83db189ad9.png)

I don't get why she acted that way. By making an holy site there she took the worst possibly decision (well, at least she enhanced already) imo, let see the alternatives:

- use the prophet to spread

There's a religious cold war going on at the border and her founder rewards spreading.

- use the prophet as inquisitor

Best done after having spread thrice, could have kept it around with 1 charge. Now my missionary is going to flip that city.

- place the holy site anywhere else near Newcastle

That 5 pop city had 3 good tiles: 2 pastures and one mine. Could have gained a good 4th tile, instead replaced the pasture only to gain 1 hammer on the GPI, not worth at this stage (later with more specialists and stables maybe...). Thanks to her founder the holy site provides 5 more tourism, not very useful on brand new that border city (if it was another founder for science or culture I would have thought she used it to fight unhappiness). Then we have the issue that with the city not following her religion the bonus is not there yet...

- Place the holy site right next to London, or even better between London and York.

Saves 4-5 turns of moving around/paying maintenance/not getting any yields out of the GP. Replaces a barebone farm that doesn't even belong to a farm triangle, or a no-triangle lumbermill. 
York could work the tile if the capital population is busy with specialists, until end game when you really want to focus the tourism in a single city.

*Disclaimer: I don't live in Newcastle nor have anything against it IRL :)

**Comment by HungryForFood** [comment]
To be fair, this is not outside of the realm of what I'd do as a human player after I have enhanced. I usually play at Emperor.

1. I'd rather not use Prophets to spread, when I can use Missionaries and Inquisitors. I only use Prophets in this manner when my religion is about to be wiped out. I'd prefer to declare war to stop Missionaries, than to use Prophets to spread.
2. I like putting GPTIs on resources to have one really good tile I can always work. This is especially nice in the late game when you start working specialists. The Pasture improvement also doesn't give much additional yields unless going Fealty, so I go for Pasture resources first, though usually Sheep first.
3. I prefer not to place on flat tiles, especially flat Grasslands, because farm triangles are really nice. I would place GPTIs on empty hills first before empty flatlands.
4. I think you should also consider that the 5 culture from the holy site is really nice at this stage of the game.

In short, perhaps not optimal, but I don't think its completely insensible.

**Comment by ilteroi** [comment]
they do have a tendency to place holy sites on resources ... seems suboptimal?

![image](https://user-images.githubusercontent.com/1358966/76872060-300aeb00-686c-11ea-9cdc-3a6ffe08ff83.png)

**Comment by LoneGazebo** [comment]
They connect resources. But not bonus ones. Might be worth excluding bonus resources, though I know some power players put GPs on sheep etc.

**Comment by randomnublet** [comment]
Pasture resources are a good tile to put a GPI on, no complaints here. Heck I do the same. In the examples above those are some pretty bad holy sites but the stone/salt tiles are still getting bonus yields from the stoneworks so it might be worth work them. Desert wheat is terrible no matter what though.

But why moving all the way from London to Newcastle, a city that doesn't have many good tiles to work anyway (a pasture at that stage was pretty good for the city, that has more population than good tiles), while the capital (especially with Way of the Pilgrim) has better multipliers for culture and tourism?

**Comment by ilteroi** [comment]
i removed the resource hookup bonus for GPTI improvements. they still connect the resource but AI won't be motivated by it.
